web/runtest.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
If the root-level jenkins build handles this, then I vote for removing this file to avoid having different adhoc techniques in different directories